lib/cli/src/Cardano/CLI.hs

```diff
@@ -1421,7 +1421,8 @@ transactionIdArgument = argumentT $ mempty
 -- | <name=STRING>
 walletNameArgument :: Parser WalletName
 walletNameArgument = argumentT $ mempty
-    <> metavar "STRING"
+    <> metavar "WALLET_NAME"
+    <> help "The name of the new wallet."
```

Comment:
"new" implies a certain context which may not fit the use-case (although we use it only for creating wallet _I think_).
I'd simply stick to "Name of the wallet" for description.

Reply:
I tried making it sound more like "here you get to choose the local name of the wallet", rather than "something that must match some existing value" — but fine, let's go for "Name of the wallet"